test/mpi/test_mpi.cc

```diff
@@ -298,6 +298,7 @@ int ucc_coll_is_rooted(ucc_coll_type_t c)
     case UCC_COLL_TYPE_ALLTOALLV:
     case UCC_COLL_TYPE_BARRIER:
     case UCC_COLL_TYPE_REDUCE_SCATTER:
+    case UCC_COLL_TYPE_REDUCE_SCATTERV:
```

Comment:
need to add UCC_COLL_TYPE_REDUCE_SCATTERV to run_all_at_team to run different datatypes and ops

Reply:
done